### PR TITLE
bugfix for hive runner log missing and partitions marked done when sq…

### PR DESCRIPTION
**Pull request by Chyler**
…l failed.

### CODE REVIEW

File: odps-data-carrier/task-scheduler/src/main/java/com/aliyun/odps/datacarrier/taskscheduler/HiveRunner.java [hunk 1]
```diff
@@ -117,6 +118,7 @@ public void run() {
 //          }
             break;
           }
+          hiveStatement.close();
```
Comment (cornmonster) [inline odps-data-carrier/task-scheduler/src/main/java/com/aliyun/odps/datacarrier/taskscheduler/HiveRunner.java]: 这里在for loop里边把statement close了，下一次循环会出错

Comment (cornmonster) [reply odps-data-carrier/task-scheduler/src/main/java/com/aliyun/odps/datacarrier/taskscheduler/HiveRunner.java]: 干脆在loop里面创建create statement吧

Comment (Chyler) [reply odps-data-carrier/task-scheduler/src/main/java/com/aliyun/odps/datacarrier/taskscheduler/HiveRunner.java]: 去掉hiveStatement.close();就ok了，后面后statement的close

**Comment by cornmonster** [comment]
+1

**Pull request merged**